README.md

```diff
@@ -64,9 +64,18 @@ err = cli.ValidateEgress(context.TODO(), "vpcSubnetID", "cloudImageID")
 
 ### Validate egress using command line
 
+Build the cli executable 
 ```shell
-AWS_ACCESS_KEY_ID=<redacted> AWS_SECRET_ACCESS_KEY=<redacted> ./osd-network-verifier egress --subnet-id subnet-0ccetestsubnet1864 --image-id=ami-0df9a9ade3c65a1c7
+make build
+```
+Execute 
+```shell
+AWS_ACCESS_KEY_ID=<redacted> AWS_SECRET_ACCESS_KEY=<redacted> ./osd-network-verifier egress --subnet-id <subnet-id> --image-id=<image-id>
```

Comment:
Instead of redacted, could place something like <key> and <secret> here?

Reply:
I'm not sure what you mean by "something like and here". It was in the existing code but I can replace it with a random string.